README.md

```diff
@@ -2,13 +2,13 @@
 
 # A Package for Exploratory Data Analysis (eda_analysis)
 
-![](https://github.com/sweber15/eda_analysis/workflows/build/badge.svg) 
+![build](https://github.com/UBC-MDS/edapython/workflows/build/badge.svg) 
 
-![codecov](https://codecov.io/gh/sweber15/eda_analysis/branch/master/graph/badge.svg)
+[![codecov](https://codecov.io/gh/UBC-MDS/edapython/branch/master/graph/badge.svg)](https://codecov.io/gh/UBC-MDS/edapython)
  
-![Release](https://github.com/sweber15/eda_analysis/workflows/Release/badge.svg)
+![Release](https://github.com/UBC-MDS/edapython/workflows/Release/badge.svg?branch=master)
 
-![Documentation Status](https://readthedocs.org/projects/edapython/badge/?version=latest)
+[![Documentation Status](https://readthedocs.org/projects/edapython/badge/?version=latest)
```

Comment:
```suggestion
![Documentation Status](https://readthedocs.org/projects/edapython/badge/?version=latest)
```
Is this an extra [?

Reply:
that was my initial thought; however, all the badges have that extra square bracket in front of it; it looks like it might be forcing a new-line per badge. 

I'm going to leave it as is.

Reply:
I mean if you go to your branch, there is an extra square bracket only before this badge.